src/engine/filters/enginefilteriir.h

```diff
@@ -90,110 +92,136 @@ class EngineFilterIIR : public EngineFilterIIRBase {
         m_doRamping = true;
     }
 
-    void setCoefs(const char* spec, double sampleRate,
-            double freq0, double freq1 = 0, int adj = 0) {
-
+    void setCoefs(const char* spec,
```

Comment:
We need to remember that bufsize includes the null-terminator, the naming reflects that already. Ok.

Switching to std::string would be a safe option. But since this is legacy C code let's keep it as is.